app/uk/gov/hmrc/nisp/views/header_nav.scala.html

```diff
@@ -21,24 +21,21 @@
 
 <div class="header-proposition">
     <div class="content">
-        
-        <nav id="proposition-menu" class="header__menu" role="navigation">
```

Comment:
just removed the <nav> element as suggested by the a11y report

Reply:
This file isn't used anywhere so can be deleted. What Nav element are you trying to remove?

Reply:
sure, 

just noticed there is the `<nav>` in the banner 
<img width="702" alt="Screenshot 2021-06-15 at 10 37 59" src="https://user-images.githubusercontent.com/27204388/122030931-1e452a00-cdc6-11eb-8453-228888b1325c.png">

This <nav> needs to be removed, not sure how, as it comes from a lib it seems, i need to take help of a UI dev :)